src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.uses_crate.txt

```diff
@@ -36,12 +36,12 @@
    22|      2|    println!("used_only_from_this_lib_crate_generic_function with {:?}", arg);
    23|      2|}
   ------------------
-  | used_crate::used_only_from_this_lib_crate_generic_function::<&str>:
+  | used_crate::used_only_from_this_lib_crate_generic_function::<alloc::vec::Vec<i32>>:
```

Comment:
I think these are just reordered. If you're generating the new coverage results on Windows, that's a common problem but it's not wrong. Sadly llvm coverage report generation does not have deterministic ordering of results. Generally they are very consistently ordered on Linux specifically, and just as consistently ordered in the opposite way on Windows. I'm not sure if that's the issue here or not.